### ISSUE TITLE
error Could not find "window._sharedData" in html response

### ISSUE THREAD

**Issue by afterdelight** [instaloader/instaloader#1555]
**Describe the bug**
Could not find "window._sharedData" in html response

**To Reproduce**
E:\downloader\instagram\insta loader>instaloader +main1.txt +stories.txt +login1.txt +commands.txt

**Expected behavior**
downlaod post

**Error messages and tracebacks**
```
Using latest stamps from C:\Users\X\AppData\Local\Instaloader\latest-stamps.ini.
Loaded session from C:\Users\X\AppData\Local\Instaloader\session-donuts_sprinkles_.
Logged in as donuts_sprinkles_.
JSON Query to jessicathomaas/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to jessicathomaas/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
jessicathomaas: Profile jessicathomaas seems to exist, but could not be loaded.
JSON Query to haraofficial/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to haraofficial/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
haraofficial: Profile haraofficial seems to exist, but could not be loaded.
JSON Query to blacaofficial/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to blacaofficial/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
blacaofficial: Profile blacaofficial seems to exist, but could not be loaded.
JSON Query to seeuxiaoroo/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to seeuxiaoroo/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
seeuxiaoroo: Profile seeuxiaoroo seems to exist, but could not be loaded.
JSON Query to chaewon.tv/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to chaewon.tv/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
chaewon.tv: Profile chaewon.tv seems to exist, but could not be loaded.
JSON Query to hiromarupee/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to hiromarupee/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
hiromarupee: Profile hiromarupee seems to exist, but could not be loaded.
JSON Query to wonyoung_daily/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to wonyoung_daily/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
wonyoung_daily: Profile wonyoung_daily seems to exist, but could not be loaded.
JSON Query to for_everyoung10/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
JSON Query to for_everyoung10/feed/: Could not find "window._sharedData" in html response. [retrying; skip with ^C]
for_everyoung10: Profile for_everyoung10 seems to exist, but could not be loaded.
Saved session to C:\Users\X\AppData\Local\Instaloader\session-donuts_sprinkles_.
```

**Instaloader version**
latest

**Additional context**

**Comment by afterdelight** [comment]
why only thumbs up????

**Comment by EnthusiastAnon** [comment]
This is already being discussed in other Issues.
It works only once if you delete session and login with password.
But then the newly created session will prevent you again, so you need to delete again and so on.
This might raise suspicion, and someone said his account got banned on the 2nd login. So be careful and wait for devs of this awesome tool to totally figure out what's happening and possibly provide an updated version that resolves this.

**Comment by afterdelight** [comment]
yes, it worked now after deleting the login session. now downloading and fapping like usual

**Comment by Showmaster3000** [comment]
Duplicate of #1553 (which has a non-saying issue title).
Please continue there.
